pkgs/by-name/pl/pluginval/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  alsa-lib,
+  gtk3,
+  freetype,
+  fontconfig,
+  xorg,
+  curl,
+  webkitgtk_4_1,
+  libsysprof-capture,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libdatrie,
+  libXdmcp,
+  libdeflate,
+  lerc,
+  xz,
+  libwebp,
+  libxkbcommon,
+  libepoxy,
+  libXtst,
+  sqlite,
+  ladspaH,
+}:
+stdenv.mkDerivation rec {
+  pname = "pluginval";
+  version = "1.0.4";
+  src = fetchFromGitHub {
+    owner = "Tracktion";
+    repo = "pluginval";
+    rev = "v${version}";
+    sha256 = "sha256-j4Lb3pcw0931o63OvTTaIm2UzvYDIjmnaCXGvKB4gwM=";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+  buildInputs = [
+    alsa-lib
+    gtk3
+    freetype
+    fontconfig
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXext
+    xorg.libXinerama
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    curl
+    webkitgtk_4_1
+    libsysprof-capture
+    pcre2
+    util-linux
+    libselinux
+    libsepol
+    libthai
+    libdatrie
+    libXdmcp
+    libdeflate
+    lerc
+    xz
+    libwebp
+    libxkbcommon
+    libepoxy
+    libXtst
+    sqlite
+    ladspaH
+  ];
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Debug"
+  ];
```

Comment:
Remove this, we don't want to build debug versions by default.

Reply:
Thanks for the review, I'm working through it as we speak.
I'm building the debug version, because that is [what the README says](https://github.com/Tracktion/pluginval?tab=readme-ov-file#installation).
I have almost no experience with debugging, but I kinda assumed that because this is a program used for debugging, you need the debug symbols.
Maybe that makes no sense at all...

Reply:
Typically you will be debugging the plugins and not the program itself.  This very case may be different though.  Can you try building the release version and see if that works for you?

Reply:
I initially built the release version, and it works fine, but I haven't used it for debugging yet, since I don't know how.
Then I noticed this in the README and I changed it to the debug version, thinking it might be useful to others.

I've asked some proper audio developers, hopefully they will reply here.

Reply:
I got a reply in a chat:
> I'd say definitely ignore that warning and follow the specific pluginval advice. Makes more sense to have pluginval in debug

So I guess I'll leave it like this until someone tells us otherwise.

Reply:
Okay then.  But maybe change this to use `lib.cmakeFeature`.